includes/event/event-capabilities.php

```diff
@@ -9,10 +9,11 @@
 use Wporg\TranslationEvents\Stats\Stats_Calculator;
 
 class Event_Capabilities {
-	private const CREATE = 'create_translation_event';
-	private const VIEW   = 'view_translation_event';
-	private const EDIT   = 'edit_translation_event';
-	private const DELETE = 'delete_translation_event';
+	private const CREATE                 = 'create_translation_event';
+	private const VIEW                   = 'view_translation_event';
+	private const EDIT                   = 'edit_translation_event';
+	private const DELETE                 = 'delete_translation_event';
+	private const MANAGE_EVENT_ATTENDEES = 'manage_translation_event_attendees';
```

Comment:
Could we call the capability `EDIT` instead of `MANAGE`? Manage makes me think of elevated permissions (e.g. super admin), but anyone who can edit the event should be able to edit the attendees. I see attendees as another property of the event (like title or start date), so to me changing attendees would be editing the event.

Additionally, because this is the `Event_Capabilities` class, we don't need the `_EVENT_`.

```suggestion
	private const EDIT_ATTENDEES = 'edit_translation_event_attendees';
```

Reply:
I agree! Thank you!